scripts/ldmx-env.sh

```diff
@@ -198,7 +198,6 @@ if hash docker &> /dev/null; then
       -e LDMX_BASE \
       -e DISPLAY=${LDMX_CONTAINER_DISPLAY}:0 \
       $_envs \
-      -v /tmp/.X11-unix:/tmp/.X11-unix \
```

Comment:
This change is necessary for anyone running on s3df. I had to use it but I can remove it.

Reply:
Ah but there is a trick you can do on s3df, which you'll only have to do once per iana node you are on (so maybe twice ever? I usually end up on  006): 
```
touch /tmp/.X11-unix
```
and that's it

Reply:
(This is one of a few small usability issues that are resolved with `denv` as well so please look at moving your workflow to using that program instead of `scripts/ldmx-env.sh`. https://ldmx-software.github.io/developing/quick-switch.html)

Reply:
@rodwyer100 please revert the changes in this file